modules/chapter2/pages/section2.adoc

```diff
@@ -0,0 +1,7 @@
+= Data Exploration
+
+This section covers Data Exploration of the Diabetes dataset that we would be using for this lab
+
+Open the 'PyTorch' workbench and clone the following Github repository https://github.com/RedHatQuickCourses/rhods-qc-apps.git
```

Comment:
I think we should highlight where to find the Git client for users that are not familiar with JupyterLab. Adding a screenshot might help too.

Reply:
Since you're also referring to the file browser below, giving a quick overview of the JupyterLab toolbar could be useful.

Reply:
Will talk to Jaime and make a note to add the JupyterLab stuff in the next iteration